### PR TITLE
Icon: Make "Show Cooldown Text" toggle work with OmniCC

### PR DESCRIPTION
**Pull request by InfusOnWoW**
Turns out that there's indeed a way to tell OmniCC that information,
as pointed via the ticket.

Also rename and inverse the option to "Hide Cooldown Text" and make it
always visible.

We used to hide the option also for the case where the user was using
Blizzard's cooldowns but with the text option not enabled. In that case
the option obviously had no effect.

But, showing the option allows aura authors to use it to disable the
cooldown text for all users.

For the same setup - Blizzard without Text -, having the option be
"Show Cooldown Text" would have been odd, as enabling it would not have
shown any text. The renaming fixes that.

Ticket-Nr: 1320

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: WeakAuras/RegionTypes/Icon.lua [hunk 1]
```diff
@@ -374,7 +374,8 @@ local function modify(parent, region, data)
   end
 
   cooldown:SetReverse(not data.inverse);
-  cooldown:SetHideCountdownNumbers(not data.cooldownTextEnabled or IsAddOnLoaded("OmniCC") or false);
+  cooldown:SetHideCountdownNumbers(data.cooldownTextDisabled);
+  cooldown.noCooldownCount = data.cooldownTextDisabled;
```
Comment (InfusOnWoW) [inline WeakAuras/RegionTypes/Icon.lua]: This is the important line

File: WeakAuras/WeakAuras.lua [hunk 2]
```diff
@@ -2765,6 +2765,14 @@ function WeakAuras.Modernize(data)
     end
   end
 
+  -- Version 4 was introduced Februar 2019 in BfA
```
Comment (emptyrivers) [inline WeakAuras/WeakAuras.lua]: Version 4 (?) was introduced in Februar?

**Comment by emptyrivers** [comment]
@InfusOnWoW did you want to reopen this? It looks like you deleted your branch unintentionally.

**Comment by InfusOnWoW** [comment]
I'll repush my branch later, I intend it for 2.12 anyway.

**Comment by emptyrivers** [comment]
Ah, ok

**Pull request closed without merging**